static/scss/components/banners.scss

```diff
@@ -54,6 +54,31 @@
     }
 }
 
+#mpp-choose-subdomain {
+    // Add a transparent border by default so its content doesn't jump around when the border appears:
+    border: 4px solid transparent;
+
+    &:target {
+        border-color: $color-informational;
+        // The height of the fixed header can vary depending on e.g. whether .micro-survey-banner
+        // is displayed, so this calculation might not result in #mpp-choose-subdomain to be
+        // anchored exactly to the top of the page, but at least it won't be hidden behind
+        // the fixed headers:
+        scroll-margin-top: calc(
+            /* .micro-survey-banner vertical padding: */
+            #{($spacing-md * 2)} +
+            /* .header-top vertical padding: */
+            #{($spacing-xl * 2)} +
+            /* .logo-wrapper height: */
+            40px +
+            /* #mpp-choose-subdomain top border: */
+            4px +
+            /* .branner-gradient-bg bottom margin: */
+            2rem
+        );
+    }
+}
```

Comment:
This feels really fragile/a magic-number. Is there a more generic way we can set this? Or can we use JavaScript to calculate this value?

Reply:
It is :/ I might be able to calculate the bounding box of the header elements and set this property in an `onclick` handler, but then I'd need to know for sure what elements all could be part of the header, since I saw at least the regular header and an NPS survey banner - do you know if there's anything else to take into account?

Reply:
We are using `position: sticky` on the header. Does that help targeting here? (We were actually using `position: fixed` before and had to listen via JS to set the main container top-margin to compensate. 

What happens when it just hits it via the `#mpp-choose-subdomain`? (with no scroll-margin-top)?

Reply:
Hmm, unfortunately I don't think it's possible to use selectors to target specific styles. I could try to iterate over all (or a set decided via some heuristics) DOM elements, inspect all their styles and calculate the heights of everything with `position: fixed`, but I expect that that would add a noticeable and annoying delay to the scroll.

Without `scroll-margin-top` (or if the calculation above results in a number that is too low), the subdomain chooser will be at the top of the viewport, hidden behind the header and, if present, the microsurvey and any other elements with `position: fixed` and `top: 0`. If the calculation above results in a number that is too high, it will be located a bit below the elements with `position: fixed` and `top: 0` (but likely to still be in view).